geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/DeltaWriter.scala

```diff
@@ -382,7 +397,7 @@ object DeltaWriter extends StrictLogging {
       }
     }
 
-    SimpleFeatureArrowIO.createFile(result, None)(iter)
+    SimpleFeatureArrowIO.createFile(result, None, iter)
```

Comment:
Is there an API change here?

Reply:
the old methods are still there but deprecated.